src/helpers/runNotificationListener.ts

```diff
@@ -0,0 +1,27 @@
+import { TokenModel } from '@/models/Token'
+import getObssContract from '@/helpers/getObssContract'
+import sendAppleNotification from '@/helpers/sendAppleNotification'
+import sendGoogleNotification from '@/helpers/sendGoogleNotification'
+
+const apnRegex = /^[a-f0-9]{64}$/
```

Comment:
Btw, if we already have a regex for this, should we use it in `env.ts`?
![image](https://user-images.githubusercontent.com/49658988/224363501-28a83a75-c330-43da-b11c-b16d945fe992.png)

Reply:
no, not related